lib/core/test/data/Cardano/Wallet/Api/HardwareWalletPostData.json

```diff
@@ -0,0 +1,45 @@
+{
+    "seed": -4365653956898380124,
+    "samples": [
+        {
+            "account_public_key": "6562643630396264366339313064643635653532306663393630306235353336353539623864363366653965313164613866633565366531633632626635653965343239343432613732653338613234383764613639393462613532376363376166636337366636323736643363393830366437333063363563646466636261",
```

Comment:
I wonder if this account public key generated is realistic :thinking:

Reply:
The account public key is 64 bytes in length, so 128 characters hex-encoded.

Reply:
indeed, I corrected the corresponding code